include/openssl/bio.h

```diff
@@ -255,7 +255,7 @@ typedef struct bio_method_st BIO_METHOD;
 const char *BIO_method_name(const BIO *b);
 int BIO_method_type(const BIO *b);
 
-typedef void bio_info_cb(BIO *, int, const char *, int, long, long);
+typedef int bio_info_cb(BIO *, int, int);
```

Comment:
here bio_info_cb does not match with the .pod above, that causes the confusion.

Reply:
Ouch! ... Need some thinking... Basically even rest of it is not kind of thing that is appropriate to do between minor releases...

Reply:
@dot-asm you asked for this warning....

Reply:
I'm not opposed to the warning, but there are different ways to resolve it... As I said I need some time for consideration, but have to step out for lunch...

Reply:
> I'm not opposed to the warning,

It's actually contrary, I appreciate it!

Reply:
I naturally meant to write "moreover" and not "it's contrary". Too little blood to my brain for the moment...

Reply:
I try to get the warning in for gcc-8.  If it gets approved in the current form
it will be enabled with -Wextra.
That means this needs back-porting to 1.1.0 at leaset
or we get into trouble with -strict-warnings build.